### DISCUSSION TITLE
英会話10/17放送分問題対策(2022年10月緊急対応)

### DISCUSSION THREAD

**Discussion by CSReviser** [CSReviser/CaptureStream#15] [General]
2022/10/17放送分「ラジオ英会話」で発生している録音が失敗する問題への緊急対応です。

※ffmpegはバージョン4.2.x以前は使えませんのでご注意ください。
※Mac版は環境がないので、動作未確認です。

<hr /><em>This discussion was created from the release <a href='https://github.com/CSReviser/CaptureStream/releases/tag/20221026'>英会話10/17放送分問題対策(2022年10月緊急対応)</a>.</em>

**Comment by toshiki-haru** [comment]
2022/10/18放送分も17日と同じ内容です

**Reply by CSReviser** [reply]
10/24、10/25放送分のラジオ英会話でも同じ問題があるようですが、緊急対応版を使えば回避できると思います。
